HW05A_script.R

```diff
@@ -0,0 +1,20 @@
+#HW05 Part A
```

Comment:
It might be a personal preference but an .Rmd would have been a little easier to review, especially having both the code and the knitted .md document. That being said, I was able to get the tables as described by executing this code.